extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -1063,3 +1063,120 @@ async def get(self, objectId: str = None):
+            - in: query
+              name: dec
+              required: true
+              schema:
+                type: float
+              description: Dec. in degrees
+            - in: query
+              name: radius
+              required: true
+              schema:
+                type: float
+              description: Max distance from specified (RA, Dec) in degrees
+          responses:
+            200:
+              description: retrieved alert(s)
+              content:
+                application/json:
+                  schema:
+                    allOf:
+                      - $ref: '#/components/schemas/Success'
+                      - type: object
+                        properties:
+                          data:
+                            type: array
+                            items:
+                              type: object
+            400:
+              content:
+                application/json:
+                  schema: Error
+        """
+        ra = self.get_query_argument("ra", None)
+        dec = self.get_query_argument("dec", None)
+        radius = self.get_query_argument("radius", None)
+        if ra is None:
+            return self.error("Missing required parameter: ra")
+        if dec is None:
+            return self.error("Missing required parameter: dec")
+        if radius is None:
+            return self.error("Missing required parameter: radius")
+        try:
+            ra = float(ra)
+            dec = float(dec)
+            radius = float(radius)
+        except ValueError:
+            return self.error("Invalid (non-float) value provided.")
+        streams = self.get_user_streams()
+
+        # allow access to public data only by default
+        selector = {1}
+
+        for stream in streams:
+            if "ztf" in stream.name.lower():
+                selector.update(set(stream.altdata.get("selector", [])))
+
+        selector = list(selector)
+
+        query = {
+            "query_type": "near",
+            "query": {
+                "max_distance": radius,
+                "distance_units": "deg",
+                "radec": {"query_coords": [ra, dec]},
+                "catalogs": {
+                    "ZTF_alerts": {
+                        "filter": {},
+                        "projection": {
+                            "objectId": 1,
+                            "candid": {"$toString": "$candid"},
+                            "candidate.ra": 1,
+                            "candidate.dec": 1,
+                            "candidate.jd": 1,
+                            "candidate.fid": 1,
+                            "candidate.magpsf": 1,
+                            "candidate.sigmapsf": 1,
+                            "candidate.drb": 1,
+                            "candidate.programid": 1,
+                            "_id": 0,
+                        },
+                    }
+                },
+            },
```

Comment:
Let's put in some noob protection (return at most a thousand objects):

```suggestion
            },
            "kwargs": {
                "limit": 1000
            }
```